Scripts/update_plist_info.sh

```diff
@@ -9,9 +9,9 @@ if [ "${PROJECT_DIR}" = "" ]; then
 fi
 
 # Capture hash & comment from last WebRTC git commit.
-cd $PROJECT_DIR/ThirdParty/WebRTC/
+cd "${PROJECT_DIR}"/ThirdParty/WebRTC/
```

Comment:
The braces are superfluous, simply adding the quotes `"$PROJECT_DIR"/ThirdParty/WebRTC/` will suffice.

Reply:
(...this, and other such issues, is also already covered in the existing PR https://github.com/signalapp/Signal-iOS/pull/4537)

Reply:
Alright, gotcha, then let me close this PR